### PR TITLE
Komga: Added README file

### PR DESCRIPTION
**Pull request by jopejoe1**
Would be nice if @gotson could go over the readme file and check if there is anything else that could be added.

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by gotson** [comment]
That's looks great, thanks!

File: src/all/komga/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Table of Content
+- [FAQ](#FAQ)
+- [Guides](#Guides)
+
+Don't find the question you are look for go check out our general FAQs and Guides over at [Extension FAQ](https://tachiyomi.org/help/faq/#extensions) or [Getting Started](https://tachiyomi.org/help/guides/getting-started/#installation)
+
+## FAQ
+
+### Why do i see no manga?
```
Comment (ghostbear) [inline src/all/komga/README.md:11]:
```suggestion
### Why do I see no manga?
```

File: src/all/komga/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Komga is a self-hosted comic/manga media server.
+
+### Where can i get more information about Komga?
```
Comment (ghostbear) [inline src/all/komga/README.md:14]:
```suggestion
### Where can I get more information about Komga?
```

File: src/all/komga/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+You can visit the [Komga](https://komga.org/) website for for more information.
+
+### The Komga extension stopped working?
+Make sure that your Komga server and extension are on the newest version.
+
+### Can i add more than one Komga server or user?
```
Comment (ghostbear) [inline src/all/komga/README.md:20]:
```suggestion
### Can I add more than one Komga server or user?
```

File: src/all/komga/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Yes, currently you can add up to 3 different Komga instances to Tachiyomi.
+
+## Guides
+
+### How do i add my Komga server to Tachiyomi?
```
Comment (ghostbear) [inline src/all/komga/README.md:25]:
```suggestion
### How do I add my Komga server to Tachiyomi?
```

File: src/all/komga/README.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+Go into the setting menu of the Komga extension and fill in your server address and login details.
```
Comment (ghostbear) [inline src/all/komga/README.md:26]:
```suggestion
Go into the settings of the Komga extension from the Extension tab in Browse and fill in your server address and login details.
```

Comment (jopejoe1) [reply src/all/komga/README.md:26]: Can't change it in this PR as its already merged would require a new PR.

Comment (CarlosEsco) [reply src/all/komga/README.md:26]: Make a new one, and combine a few readmes.  I'll let it simmer for a day so people can review then merge.

**Pull request merged**